src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -38,6 +40,7 @@ class HttpKernel implements HttpKernelInterface, TerminableInterface
     protected $dispatcher;
     protected $resolver;
     protected $requestStack;
+    protected $httpFoundationFactory;
```

Comment:
should be private (we always start by making everything private until we need to provide an extension point)